SwiftFest/Models/Sponsor.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Sponsor.swift
+//  SwiftFest
+//
+//  Created by Matthew Dias on 7/14/19.
+//  Copyright © 2019 Sean Olszewski. All rights reserved.
```

Comment:
Can you update the copyright? It should be all open source here.

Reply:
I'll updated it once the build is made